content/en/docs/contributing/e2e.md

```diff
@@ -30,18 +30,52 @@ run the tests:
   not required. For non-Linux hosts (i.e. OSX), you will need to ensure you have
   a relatively new version of `kubectl` available on your PATH.
 
+- `kind`: We use kind to provision a Kubernetes cluster.
+
 - An internet connection: tests require access to DNS, and optionally CloudFlare
   APIs (if a CloudFlare API token is provided).
 
 Bazel, Docker and `kubectl` should be installed through your preferred means.
 
+## Set up End-to-End Tests
+
+You need to have a Kind cluster running, if you don't have one set up you can set one up using:
+```bash
+$ ./devel/setup-e2e-tests.sh
+```
+
+Once you have one set up you need to install all dependencies in the cluster using:
+```bash
+$ ./devel/create.sh
+```
```

Comment:
This should be `./devel/setup-e2e-deps.sh` https://github.com/jetstack/cert-manager/blob/master/devel/setup-e2e-deps.sh